src/commands/login.rs

```diff
@@ -0,0 +1,55 @@
+use crate::commands::OroCommand;
+use async_trait::async_trait;
+use clap::Args;
+use directories::ProjectDirs;
+use kdl::KdlDocument;
+use miette::{IntoDiagnostic, Result};
+use oro_client::login::AuthType;
+use oro_npm_account::config::{self, Credentials};
+use oro_npm_account::login::login;
+use url::Url;
+
+/// Log in to the registry.
+#[derive(Debug, Args)]
+pub struct LoginCmd {
+    #[arg(from_global)]
```

Comment:
the `npm login` command also accepts a scope, in which case it will add a scoped registry config to .npmconfig.

In our case, `oro login --scope @mycompany --registry "https://my.company.net" would result in a config like:

```kdl
options {
  scoped-registries {
    @mycompany "https://my.company.net"
  }
  auth {
    "https://my.company.net" token="deadbeef"
  }
}
```